doc/dev/development.rst

```diff
@@ -166,10 +166,62 @@ To test PypeIt using the data from the Google TeamDrive:
 
    .. warning::
         
-        The current script executes 31 tests.  These tests are mostly
+        The current script executes 89 tests.  These tests are mostly
         reductions of example data, but they also include fluxing,
         flexure, and coadding tests.  The execution time is system
-        dependent, but you should expect it to take approx. 12 hours.
+        dependent, but you should expect it to take approx. 14 hours.
+
+   The test suite can be run in parallel using the -t option.
+
+   .. code-block:: bash
+
+        ./pypeit_test -t 2 develop
+
+   The above runs tests using two processes run from parallel threads. The number of
+   threads that can be used reliably depends on the amount of memory available. The
+   below table shows the worst case amount of memory used per number of threads.
+
+   +--------------+-------------+
+   | # of threads | Memory used |
```

Comment:
One quick point of clarification, maybe edit the heading of the second column to be "Memory used (GB)".